### PR TITLE
finished es6-classes

### PR DESCRIPTION
**Pull request by kyle-foral**
image
image
image

### CODE REVIEW

File: es6-classes/square.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) { super(); }
```
Comment (shawnkost) [inline es6-classes/square.js:5]: You should also be defining `this.width` with the value that gets passed in from the `width` parameter

File: es6-classes/square.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  describe() {
+    return 'This is the width ' + this.width;
+  }
+}
+
+const square = new Square();
```
Comment (shawnkost) [inline es6-classes/square.js:11]: When you instantiate `new Square()` you should be passing in an argument representing the `width`

File: es6-classes/circle.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(radius) {
+    super(area, perimeter);
```
Comment (shawnkost) [inline es6-classes/circle.js:6]: `area` and `perimeter` haven't been defined. You should be passing in a number here, utilizing your `radius` parameter.

File: es6-classes/shape.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    this.area = area;
+    this.perimeter = perimeter;
+  }
+
+  describe() {
+    return 'This is the area and perimeter ' + this.area + ' ' + this.perimeter;
+  }
+}
+
+const shape = new Shape();
```
Comment (shawnkost) [inline es6-classes/shape.js:14]: You should be passing in 2 arguments here. One representing the `area` & the other `perimeter`

File: es6-classes/square.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) {
+    super(area, perimeter);
```
Comment (shawnkost) [inline es6-classes/square.js:6]: `area` and `perimeter` haven't been defined. You should be passing in a number here, utilizing your `width` parameter.

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes here Kyle!

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**